frontend/src/components/organisms/ManageAttendees.tsx

```diff
@@ -44,7 +48,34 @@ type FormValues = {
   endTime: string;
 };
 
-interface ManageAttendeesProps {}
+interface ManageAttendeesProps { }
+
+
+const getEventIdFromUrl = (url: string = window.location.href): string | null => {
+  const regex = /\/events\/([^\/]+)/;
+  const match = url.match(regex);
+  return match ? match[1] : null;
+};
+
+const handleStatusChange = async (userId: string, newValue: string) => {
+  const eventId = getEventIdFromUrl(); // Retrieve event ID from URL
+  if (!eventId) {
+    console.error("Event ID not found in URL");
+    return;
+  }
+
+  try {
+    // Make a PUT request to update the user
+    const response = await api.put(`/events/${eventId}/users/${userId}`, {
+      status: newValue // Only update the status field
+    }); console.log("success")
+  } catch (error) {
+    console.error("Error updating user status:", error);
+  }
+};
+
```

Comment:
Move this function into the component it is "serving". Better code structure

Reply:
Additionally, wrap it in a Tanstack router mutation - so that you can automatically invalidate the cache and the relevant information is re-populated without the need for refreshing the page! It's pretty neat!